pkgs/by-name/im/immich-go/package.nix

```
@@ -7,7 +7,21 @@ buildGoModule rec {
     owner = "simulot";
     repo = "immich-go";
     rev = "${version}";
-    hash = "sha256-zYqPPLDfBx4FLvZIo5E6nAeIiFfBCLI00xLieXFkMxs=";
+    hash = "sha256-MOFJnUjGocfHvO+8S/tcNEnXZBJdvk2oXU+gRNj3d1U=";
+
+    # Inspired by: https://github.com/NixOS/nixpkgs/blob/f2d7a289c5a5ece8521dd082b81ac7e4a57c2c5c/pkgs/applications/graphics/pdfcpu/default.nix#L20-L32
+    # The trick used here is to write that information into files in `src`'s
+    # `$out`, and then read them into the `ldflags`. We also delete the `.git`
+    # directories in `src`'s $out afterwards, imitating what's done if
+    # `leaveDotGit = false;`
+    # %H -> commit hash; %cd -> committer date
```

This is mostly stylistic. Feel free to implement or not.

```suggestion
    # The intention here is to write the information into files in the `src`'s
    # `$out`, and use them later in other phases (in this case `preBuild`). 
    # In order to keep determinism, we also delete the `.git` directory
    # afterwards, imitating the default behavior of `leaveDotGit = false`.
    # More info about git log format can be found at `git-log(1)` manpage.
```

---

Sure, thank you for the rewording :) I was mostly trying to 're-use' the original message. Next time I will try to provide more context :+1:
Btw, if you don't mind asking @AndersonTorres, where can I learn how to _correctly_ accept the suggested changes with the nixpkgs style? I've used the GitHub interface to commit the suggestion directly, pulled the branch locally and squashed the commit, but it triggers an unnecessary CI run. :/
I couldn't find anything about it in the contributor's documentation, but I could've easily missed something.